cmd/main.go

```diff
@@ -1,43 +1,66 @@
+// Package main is the entry point of the application.
 package main
 
 import (
 	"fmt"
 	"github.com/isdzulqor/donation-hub/internal/core/service/project"
 	"github.com/isdzulqor/donation-hub/internal/core/service/user"
+	encryption "github.com/isdzulqor/donation-hub/internal/driven/encryption/jwt"
 	"github.com/isdzulqor/donation-hub/internal/driven/storage/mysql/projectstr"
 	"github.com/isdzulqor/donation-hub/internal/driven/storage/mysql/userstr"
 	"github.com/jmoiron/sqlx"
 	"log"
 	"net/http"
+	"os"
 
 	"github.com/isdzulqor/donation-hub/internal/driver/rest"
+	"github.com/joho/godotenv"
 )
 
+// main is the main function of the application.
+// It sets up the database connection, initializes the services and starts the HTTP server.
 func main() {
-	connectionString := "root:@tcp(localhost:3306)/donation_hub"
+	err := godotenv.Load()
+	if err != nil {
+		panic(err)
+	}
+
+	// Database connection string
+	connectionString := getDatabaseConnectionString()
+
+	// Connect to the database
 	db, err := ConnectToDatabase(connectionString)
 	if err != nil {
-		log.Fatalf("Failed to connect to database: %v", err)
+		panic(err)
```

Comment:
Use log.Fatal instead of panic. Give the error mesg also to provide context. So, it's easier to debug.